src/components/Messages/Message.tsx

```diff
@@ -15,10 +16,11 @@ import CodeBlock from "@/components/Messages/CodeBlock";
 const hasAttachment = (message: UserMessage) => {
   return (
     message.screenshot ||
-    message.clipboardText ||
+    message.clipboard_text ||
     message.window ||
     message.fileTitle ||
```

Comment:
This is nitpicky but if we're using snake case for `clipboard_text`, `image_url`, etc. can you change `fileTitle` to `file_title`.

Reply:
Done!